libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -246,6 +265,18 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return false;
     }
 
+    @Override
+    public void onRedoEnabled() {
```

Comment:
Both `onRedoEnabled()` and `onUndoEnabled()` are called with every keystroke, which seems inefficient. Shouldn't these only be triggered when the enabled state of undo/redo changes?